test/integration/fixtures/blocks/core__table.html

```diff
@@ -1,3 +1,3 @@
-<!-- wp:core/table -->
+<!-- wp:core/table {"hasFixedLayout":false} -->
```

Comment:
These fixtures should reflect the current default state unless they target a specific use case. I think the better approach is to update the fixture e.g. markup etc.